cip.go

```diff
@@ -337,7 +337,12 @@ func main() {
 
 	// Check the pull request
 	if *dryRunPtr {
-		err = sc.RunChecks(promotionEdges, []reg.PreCheck{})
+		err = sc.RunChecks(promotionEdges, []reg.PreCheck{
+			reg.ImageRemovalCheck{
```

Comment:
You should instead create a helper function that returns the correct ImageRemovalCheck{} structure. Something like `func mkRealImageRemovalCheck(...) reg.ImageRemovalCheck { ... }`. The reason is because we should not just do things ad-hoc if we can avoid it.